test/cql-pytest/test_secondary_index.py

```diff
@@ -100,3 +100,15 @@ def index_used(query, index_name):
         # Local indexes are still preferred over global ones, if they can be used
         index_used(f"SELECT * FROM {table} WHERE p = 1 and v1 = 1 and v3 = 2 and v2 = 2 allow filtering", "my_v2_idx")
         index_used(f"SELECT * FROM {table} WHERE p = 1 and v2 = 1 and v1 = 2 allow filtering", "my_v2_idx")
+
+@pytest.mark.xfail(reason="Creating a maliciously named table breaks secondary index creation mechanism. Issue #8620")
```

Comment:
Please add this nice long explanation as a comment, not just as xfail reason. We will want all this information even after we fix this bug.

Reply:
Note that for a scylla_only test, I would like to see an even longer comment what this is testing and why. Why do we *want* Scylla to fail in a way that Cassandra doesn't? Is it because we have no other choice, or what?

Reply:
Sure thing, I'll add a longer explanation in v2.